rest-service/manager_rest/test/endpoints/test_tenants.py

```diff
@@ -0,0 +1,85 @@
+#########
+# Copyright (c) 2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+from mock import patch
+from manager_rest import constants, config
```

Comment:
Nitpicking, I see you have done mostly alphabetical order minus the last one. Let's still separate I think to sections, but I like the alphabetical order